src/Apollo_Client/core/Apollo_Client__Core_WatchQueryOptions.re

```diff
@@ -0,0 +1,32 @@
+module ErrorPolicy = {
+  type t =
+    | None
+    | Ignore
+    | All;
+
+  let toRaw =
+    fun
+    | None => "none"
+    | Ignore => "ignore"
+    | All => "all";
+};
```

Comment:
I usually prefer to use bs.deriving for this
https://bucklescript.github.io/docs/en/generate-converters-accessors#usage-5

Reply:
Nice! I prefer this as well, but for some reason I thought `bs.deriving` only worked on polymorphic variants and that this was an intentional style decision before.

Reply:
Well, nevermind, I got excited and misread the docs. Still need polymorphic variants, but it sounds like that's the direction we want to go, so I'll switch to that.